src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private int side;
+    private String color = getColor();
+
+    public Square(int side, String color) {
+        this.side = side;
+        this.color = color;
+    }
+
+    @Override
+    public double areaCalculator() {
+        return (side * side);
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure square, area: " + areaCalculator()
+                + ", side: " + side
+                + ", color: " + color);
```

Comment:
Magic numbers should be avoided. In this case, you could define a constant for the 'Figure square' string that is being used to print the type of figure.